### ISSUE TITLE
Invitation class attributes

### ISSUE THREAD

**Issue by fimtow** [AJOthmane/Projet-jee-2.0-#5]
![image](https://user-images.githubusercontent.com/62258310/121092279-34118880-c7e3-11eb-9702-b48c606c62b5.png)
@AJOthmane What's the post attribute for? I dont see any use for it 😕

**Comment by AJOthmane** [comment]
![image](https://user-images.githubusercontent.com/76187577/121092647-cc0f7200-c7e3-11eb-84f7-4cdc1b15ace8.png)
because we have a foreign key to to post table

**Comment by fimtow** [comment]
![image](https://user-images.githubusercontent.com/62258310/121096405-0f6cdf00-c7ea-11eb-8c8e-b88b66ee8a53.png)

But why we dont have a post attribute in text and image content 🤔. Either way, I dont see a use for the post attribute, since post table have already a foreign key to the content.

**Comment by AJOthmane** [comment]
I agree with you, I think we did a mistake in our database. what do you suggest  ? should we delete it?

**Comment by fimtow** [comment]
I think we should delete the post attribute from Invitation. I didn't see any use for it 🤔
